apps/src/templates/codeDocs/ParametersTable.jsx

```diff
@@ -31,7 +31,11 @@ const descriptionFormatter = description => {
   }
 };
 
-export default function ParametersTable({parameters}) {
+export default function ParametersTable({
+  parameters,
+  programmingEnvironmentLanguage
+}) {
+  const hideRequiredColumn = programmingEnvironmentLanguage === 'java';
```

Comment:
(for the future) I'm wondering two things about this logic:
1) Should this kind of logic be on the front-end or the back-end?
2) It seems not great to hard code the language name here.  Is there a better way to associate this attribute with the language (e.g. as a property of ProgrammingEnvironmentLanguage?)?

Reply:
hm I'm not sure if it's better to put this kind of logic on the front-end or the back-end. I think I slightly prefer it on the front end as it's about a specific front end style, but I don't have a strong opinion at all.

I'm not 100% sure what you're saying for number 2? I'm definitely happy to abstract this out in some way, especially as this might need to eventually include multiple languages